Assets/Gothic-UnZENity-Core/Scripts/Manager/GUZBootstrapper.cs

```diff
@@ -6,127 +6,47 @@
 using GUZ.Core.Debugging;
 using GUZ.Core.Globals;
 using GUZ.Core.Manager.Settings;
-using GUZ.Core.Util;
 using GUZ.Core.Vm;
-using UnityEngine;
+using GUZ.Core;
 using ZenKit;
 using ZenKit.Daedalus;
 using Debug = UnityEngine.Debug;
-using Logger = ZenKit.Logger;
 
 namespace GUZ.Core.Manager
 {
-    public class GUZBootstrapper : SingletonBehaviour<GUZBootstrapper>
+    public class GUZBootstrapper
     {
-        private bool isBootstrapped;
-        public GameObject invalidInstallationDirMessage;
-
-        private void Start()
-        {
-            Logger.Set(FeatureFlags.I.zenKitLogLevel, ZenKitLoggerCallback);
-
-            // Just in case we forgot to disable it in scene view. ;-)
-            invalidInstallationDirMessage.SetActive(false);
-        }
-
-        private void OnApplicationQuit()
+        public static void OnApplicationQuit()
         {
             GameData.Dispose();
-            AssetCache.Dispose();
+            VmInstanceManager.Dispose();
             TextureCache.Dispose();
             LookupCache.Dispose();
-            PrefabCache.Dispose();
             MorphMeshCache.Dispose();
         }
-
-        private void Update()
-        {
-            // Load after Start() so that other MonoBehaviours can subscribe to DaedalusVM events.
-            if (isBootstrapped)
-                return;
-            isBootstrapped = true;
-
-            var g1Dir = SettingsManager.GameSettings.GothicIPath;
-
-            if (SettingsManager.CheckIfGothic1InstallationExists())
-            {
-                BootGothicUnZENity(g1Dir);
-                
-#pragma warning disable CS4014 // It's intended, that this async call is not awaited.
-                GUZSceneManager.I.LoadStartupScenes();
-#pragma warning restore CS4014
-            }
-            else
-            {
-                // Show the startup config message.
-                invalidInstallationDirMessage.SetActive(true);
-            }
-        }
```

Comment:
Thanks a lot for removing this one. I actually implemented this and hated it at the same time. 🤣